server.js

```diff
@@ -37,6 +37,7 @@ app.get("/api/:tenantId/parole", api.parole);
 app.get("/api/:tenantId/prison", api.prison);
 app.get("/api/:tenantId/probation", api.probation);
 app.get("/api/:tenantId/sentencing", api.sentencing);
+app.get("/api/:tenantId/race", api.race);
```

Comment:
Nit: let's keep with the consistently alphabetical ordering on the server side.

Reply:
Same goes for the other code in the server here (I recognize that this overly explicit API structure is not ideal in the long-run, believe me)